packages/components/src/Modal/GenericModal/subcomponents/ModalFooter/ModalFooter.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import classnames from "classnames"
+import { Button, ButtonProps } from "~components/Button"
+import { useMediaQueries } from "~utils/useMediaQueries"
+import { ModalSection } from "../ModalSection/ModalSection"
+import styles from "./ModalFooter.module.scss"
+
+type ActionsVariantProps = "context" | "inputEdit"
+
+export type ModalFooterProps = Readonly<{
+  /**
+   * We have a special case for the InformationModal when it has an image.
+   * Since this modal may have an image on the right side the actions might look disconected from the content.
+   * So for this instance we need to flip the order of the actions so that the primary
+   * action is anchored to the left edge of the modal.
+   * For this rare instance added the variant prop as optional to update the order of action buttons.
+   */
+  variant?: ActionsVariantProps
+  unpadded?: boolean
+  actions: ButtonProps[]
+  appearance?: "primary" | "destructive"
+  automationId?: string
+  alignStart?: boolean
+}>
+
+export const ModalFooter = (props: ModalFooterProps): JSX.Element => {
+  const {
+    unpadded,
+    actions,
+    appearance = "primary",
+    automationId,
+    variant,
+  } = props
```

Comment:
I question why most the props are destructured and some aren't. Given it's a migration though, fix only if you feel like it.